lib/mobile_app_backend_web/channels/vehicles_for_route_channel.ex

```diff
@@ -15,12 +15,34 @@ defmodule MobileAppBackendWeb.VehiclesForRouteChannel do
 
       {:ok, vehicle_data} = MBTAV3API.Stream.StaticInstance.subscribe("vehicles")
 
-      vehicle_data = filter_data(vehicle_data, route_id, direction_id)
+      vehicle_data = filter_data(vehicle_data, [route_id], direction_id)
 
       {:ok, vehicle_data,
        assign(socket,
          data: vehicle_data,
-         route_id: route_id,
+         route_ids: [route_id],
+         direction_id: direction_id,
+         throttler: throttler
+       )}
+    else
+      :error -> {:error, %{code: :missing_param}}
+    end
+  end
+
+  @impl true
+  def join("vehicles:routes:" <> topic_param_concat, _payload, socket) do
```

Comment:
> Why a separate topic rather than accepting a list in `route_id` in the existing topic?

@boringcactus  @KaylaBrady and I talked in the office yesterday, and we decided that in general we should move towards putting params into the topic names, to make it possible to connect to the same channel with different params if we need to.